felix/bpf/tc/attach.go

```diff
@@ -84,16 +84,20 @@ func (ap *AttachPoint) loadObject(ipVer int, file string) (*libbpf.Obj, error) {
 		// In case of global variables, libbpf creates an internal map <prog_name>.rodata
 		// The values are read only for the BPF programs, but can be set to a value from
 		// userspace before the program is loaded.
+		mapName := m.Name()
 		if m.IsMapInternal() {
+			if strings.HasPrefix(mapName, ".rodata") {
```

Comment:
What other maps are internal now? Would this help with movind strings from code to .rodata?

Reply:
If you remember, we used to get unrecognized section .rodata warning. Now they have defined these as separate maps (BPF_MAP_TYPE_ARRAY). These maps along with the global data is marked as RODATA only difference being global data maps are named as <map_name>.<rodata> whereas these maps are named .rodata.1.1. Hence we need to skip these maps. I am sure about string handling. Have to check how that works.